### DISCUSSION TITLE
Group Order or sell PCBs

### DISCUSSION THREAD

**Discussion by TheMBeat** [StuckAtPrototype/Racer#11] [General]
Any Plan to order assembled PCB and sell them or create a group order?

**Comment by StuckAtPrototype** [comment]
sorry, no plan right now. Id imagine you could get them assembled in single quantities with jlpcb or similar. but... you can always get the boards and hand assemble them! :) I did that for years. a good magnifying glass and some steady hands

**Reply by markus7h** [reply]
I justed checked at JLC and others  how much an assembled PCB would be. For 5 PCB JCL calculate approx. $80 for the parts and additional $11 for the PCB's. But a Pick&Place file is missing. From what I saw in the files, it could be generated from kicad, but this is beyond my knowledge. In addition I needed to guess 2 parts, that couldn't be identified. That has been the USB-C connector and the 2 pin power connector for the battery. 

I took all of that into consideration and put my efforts on hold, because of my lack of experience. It would be great if someone could configure at JLC and share a checked PCB with assembly configuration, may be a joined order can be made (I am from Germany).

**Comment by Ivy-Mo** [comment]
JLC is a good choice for single quantity test boards, but for mature designs with higher quality requirements, 10pcs or more I recommend WellerPCB. Shipping to Germany is about 36.4USD (1kg PCB&PCBA)

**Reply by blubbsy** [reply]
i think for this project PCBWay, JLC, whatever are more than sufficient.

**Comment by csuno** [comment]
... i dont have "steady hands" ;/ i would like to see some preassambled boards (i am from germany, too!)

**Reply by Ivy-Mo** [reply]
![Uploading PCBsample-1.png…]()

**Comment by Ivy-Mo** [comment]
I can't send pictures, could you give me your email?

**Reply by PB-Blink** [reply]
Yes, Thanks @Ivy-Mo , i meant: like to "see" orderable boards ;) Sorry.
I try to send you my mail, anyway! Thanks

**Reply by Ivy-Mo** [reply]
It's my misunderstanding lol, the finished boards always better, if you need to order PCB & PCBA, I'm more than welcome! Thanks :D

**Reply by Quas7** [reply]
Germany here as well... I would chim in for the PCBA group order for the next design iteration (D+,D- routing idea, etc.). :)

**Reply by TheMBeat** [reply]
Me too

**Reply by markus7h** [reply]
Is someone collecting orders to do a group order? I am interested too.

**Reply by dehlen** [reply]
I would be interested as well and am from Germany 😁

**Reply by goldjunge91** [reply]
Hi, 👋 

I started trying to get the pull requests merged in a fork and I'm waiting for an update on the repo. However, I don't want to wait much longer and will try to order it in the next few weeks/months.

**Comment by H4rry252** [comment]
I don't have steady hands either and my soldering experience is very limited. I'm also from Germany and I'm interested.

**Comment by blubbsy** [comment]
Am i seeing it right, that the BOM-Cost are approximetely 35€ per car? 

~10€ for components
~6€ the battery
~18€ the motors

Maybe a recommended component supplier would be good to tell in the project (e.g. mouser, digikey, rs-online...) where at least the components are known to be available and to save shipping cost.

**Reply by M4xTheMan** [reply]
I'd be interested too. I'm located in Germany. Unfortunately, I'm not able to organize something like that at the moment. Haven't further looked into ordering it yet in general.
